chevah/keycert/ssh.py

```diff
@@ -690,6 +697,8 @@ def _fromString_PUBLIC_OPENSSH(cls, data):
         @raises BadKeyError: if the blob type is unknown.
         """
         blob = base64.decodestring(data.split()[1])
+        if cls._guessStringType(blob) != 'blob':
+            raise BadKeyError("Fail to parse OpenSSH key content.")
```

Comment:
You will get a nice 'OpenSSH key' error only for this particular case.

what to you say about having something like this?
```
try:
    return cls._fromString_BLOB(blob)
except BadKeyError as error:
    raise BadKeyError('OpenSSH key error. %s' % (error.message'))
```